packages/react-native-macos-init/src/cli.ts

```diff
@@ -203,6 +208,26 @@ async function getLatestMatchingReactNativeMacOSVersion(
   }
 }
 
+async function updateGitIgnore(cwd: string) {
+  try {
+    const gitIgnorePath = path.join(cwd, '.gitignore');
+
+    const gitIgnore = await readFile(gitIgnorePath)
+      .then(buf => buf.toString())
+      .catch(() => null);
+
+    if (!gitIgnore || !gitIgnore.split('\n').includes(MACOS_POD)) {
+      await writeFile(
+        gitIgnorePath,
+        gitIgnore ? `${gitIgnore}\n${MACOS_POD}` : MACOS_POD,
+      );
+      console.log(chalk.green(`updated .gitignore for macos projects`));
+    }
+  } catch (error) {
+    // ignore non-critical error
+  }
+}
```

Comment:
I feel like we don’t really need to worry about it already existing in the ignore file. Most likely it’s a new project, so it won’t have it, and if it does already have it, it’s a very minor issue that won’t break things (I guess?). So I think this can be simplified to just append to the file.

Reply:
I agree that it's a minor issue. However I wouldn't think it's unnecessary (if I simply append the line, I'd feel like someone else would throw in a PR  to check if the line already exists or not)
Is there a downside of having whether the line exists or not?

Reply:
Less code is best code 😄 I’m fine merging this as-is, though 👍

Reply:
It be simpler to just add a .gitignore template file to `local-cli/generator-macos/templates/macos`.   That's what `react-native init` does for iOS.   Just add a `_gitignore` template similar to the one in `/template` (which is in this fork as well as upstream) and then the code change to copy and rename the template from `_gitignore` to `.gitignore`